doc/submitter_guide.md

```diff
@@ -312,6 +312,41 @@ Date/Time Format][datetime_format].
 
 Example: `"2020-08-14T23:41:54+00:00"`
 
+#### Issues
+
+##### `build_valid`
+The status to assign to incident builds
+
+Example: `false`
+
+##### `test_status`
+The Status to assign to incident tests, it can be one of the following
+
+* `ERROR` - the test is faulty, the status of the tested code is unknown.
+* `FAIL` - the test has failed, the tested code is faulty.
+* `PASS` - the test has passed, the tested code is correct.
+* `DONE` - the test has finished successfully, the status of the tested code
+  is unknown.
```

Comment:
This is an incomplete list of possible status strings. It might be better to just refer to the description of the test `status` field above, as they're the same type.

Reply:
@spbnick i reffered to the test:status section, the available status strings there are: Error, Fail, Pass, Done, Skip. That was what i also included